types/react-toggle/react-toggle-tests.tsx

```diff
@@ -1,10 +1,36 @@
 import * as React from "react";
-import Toggle from "react-toggle";
+import Toggle, { ToggleIcons } from "react-toggle";
 
 class Test extends React.Component {
+    handleEvent = (e: any) => {};
```

Comment:
not sure if typing this as `any` makes it much of a test...

`e: React.SyntheticEvent<any>`?